jekyll/_cci2/high-uid-error.md

```diff
@@ -64,46 +61,39 @@ you must update the files' UID/GID
 and re-create the image.
 
 If you are not the image maintainer, congratulations:
```

Comment:
> If you are not the image maintainer, congratulations:

I get the sentiment but not sure it's necessary in our docs 😆

Reply:
Yeah I dithered about that one but you are right!